packages/backend/src/config/app-env-vars/m365.ts

```diff
@@ -81,6 +81,25 @@ export const m365TenantInfo = Object.freeze({
     : {}),
 })
 
+export const m365RateLimits = Object.freeze({
```

Comment:
On 2nd thoughts.. maybe this should be defined in the `rate-limiter.ts` file. will makechange in a separate PR to prevent rebase hell but approach remains the same